shared/constants/index.ts

```diff
@@ -1 +1 @@
-export const FILE_SIZE_LIMIT = 62914560;
+export const FILE_SIZE_LIMIT = 104857600;
```

Comment:
C'est pas plus simple de mettre le calcul plutot qu'un magic number? 

```suggestion
const megaByte = 1024**2;
export const FILE_SIZE_LIMIT = 100 * megaByte;
```